bindings/matrix-sdk-crypto-ffi/src/machine.rs

```diff
@@ -930,6 +932,8 @@ impl OlmMachine {
                     },
                 }
             }
+            // Should not happen
+            _ => panic!("Unsupported algorithm in room"),
```

Comment:
Will really never happen because if not megolm the `decrypt_room_event` have already failed